lib/addStyles.js

```diff
@@ -28,9 +28,19 @@ var getElement = (function (fn) {
 
 	return function(selector) {
 		if (typeof memo[selector] === "undefined") {
-			memo[selector] = fn.call(this, selector);
+			var styleTarget = fn.call(this, selector);
+			// Special case to return head of iframe instead of iframe itself
+			try {
+				if (styleTarget instanceof window.HTMLIFrameElement) {
```

Comment:
Do it not make more sense to check `instanceof` first? As it stands right now, we are going to execute that try/catch block every time regardless. When in reality, it only needs to execute if `styleTarget instanceof window.HTMLIFrameElement`